messages/helpers.go

```diff
@@ -58,6 +59,18 @@ func ExtractProposal(proposalMessage *proto.Message) []byte {
 	return preprepareData.PreprepareData.Proposal
 }
 
+// ExtractEthereumBlock extracts the EB (EthereumBlock) from the passed in message
+func ExtractEthereumBlock(proposalMessage *proto.Message) []byte {
+	if proposalMessage.Type != proto.MessageType_PREPREPARE {
+		return nil
+	}
+
+	preprepareData, _ := proposalMessage.Payload.(*proto.Message_PreprepareData)
+	proposal := preprepareData.PreprepareData.Proposal
+
+	return proposal.GetEthereumBlock()
+}
```

Comment:
This can be replaced by:
```suggestion
func ExtractEthereumBlock(proposalMessage *proto.Message) []byte {
	proposal := ExtractProposal(proposalMessage)

	return proposal.GetEthereumBlock()
}
```